### PR TITLE
Combines gNSI.ssh with gNSI.console

### PR DESCRIPTION
**Pull request by tomek-US**

### CODE REVIEW

File: credentialz/credentialz.proto [hunk 1]
```diff
@@ -0,0 +1,605 @@
...
+  // January 1st, 1970 00:00:00 GMT.
+  uint64 created_on = 4;
+}
+
+message RotateAuthorizedUsersResponse {
+}
+
+message RotatePasswordRequest {
+  message Account {
+    message Password {
```
Comment (marcushines) [inline credentialz/credentialz.proto:479]: i don't like embeding a embedded message can we move that up?

Comment (tomek-US) [reply credentialz/credentialz.proto]: Done.

**Review by marcushines — APPROVED** [review]

File: credentialz/gnsi-credentialz.yang [hunk 2]
```diff
@@ -108,6 +127,7 @@ module gnsi-ssh {
         description
             "A system role credentials freshness information.";
 
+        uses user-console-credentials-version;
         uses user-ssh-credentials-version;
     }
     deviation  "/oc-sys:system/oc-sys:aaa/oc-sys:authentication/oc-sys:users/" +
```
Comment (robshakir) [inline credentialz/gnsi-credentialz.yang:133]: This is a nice idea -- I like the idea that there is an explicit expectation set as to not allowing this via gNMI when gNSI is used.

File: credentialz/gnsi-credentialz.yang [hunk 3]
```diff
@@ -95,6 +94,26 @@ module gnsi-ssh {
...
+    grouping user-console-credentials-version {
+        description
+            "System role credentials freshness-related data.";
+
+        leaf password-version {
+            type version;
+            description
+                "The version of the password that is currently used to
+                authenticate this user account.";
+        }
```
Comment (robshakir) [inline credentialz/gnsi-credentialz.yang:108]: (nit) blank line between these two `leaf` definitions would match the general style in OC models.

File: credentialz/credentialz.proto [hunk 4]
```diff
@@ -0,0 +1,605 @@
...
+message RotateAccountCredentialsRequest {
+  oneof request {
+    AuthorizedKeysRequest credential = 1;
+    AuthorizedUsersRequest user = 2;
+    PasswordRequest password = 3;
+    FinalizeRequest finalize = 4;
+  }
+}
+
+// RotateAccountCredentialsResponse is the response to
```
Comment (robshakir) [inline credentialz/credentialz.proto:211]: (nit) since you don't have these Go-style comments for each message, you could remove this comment.

**Review by robshakir — APPROVED** [review]
LGTM, thanks @tomek-US -- suggestions are small and editorial in nature.

**Pull request merged**